cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -125,6 +125,19 @@ spec:
       for: 40s
       labels:
         severity: critical
+    - alert: CephOSDFlapping
+      annotations:
+        description: Storage device {{ $labels.ceph_daemon }} has restarted 5 times
+          in last 5 minutes. Please check the device and ceph status to find out the
+          cause.
+        message: OCS storage device flapping.
+        severity_level: error
+        storage_type: ceph
+      expr: |
+        changes(ceph_osd_up[5m]) >= 10
```

Comment:
Is there a reason why we chose 5 times in 5 mins as the threshold?

Reply:
Yes. Got this value from the Ceph team based on past data.